docs/getting-started/installing/troubleshooting.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Troubleshooting
+---
+
+# Services don't start
+
+## Symptoms
+
+1. `kontena node ls` show nodes in a __OK__ status, but services are not starting.
+2. `kontena grid logs` show execeptions that looks like the one below
```

Comment:
The actual symptom here is that the `kontena-etcd` is broken, and `kontena etcd health` is the best way to diagnose this.